### PR TITLE
[11.0][ADD] stock_orderpoint_mrp_link

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
This module adds to manufacturing order a direct link to the reordering rules that created it. In chained moves, the reordering rule is propagated from stock moves to the manufacturing order.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Functional and technical review :+1:

File: stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (rousseldenis) [inline stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py]: @hveficent Mixing LGPL and AGPL licenses. Could you clarify ?

Comment (HviorForgeFlow) [reply stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py]: It should have been LGPL, good catch :smile:

**Comment by LoisRForgeFlow** [comment]
@rousseldenis could you update your review?

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**